drivers/hmis/spec/factories/hmis/files.rb

```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
+  factory :file, class: 'Hmis::File' do
```

Comment:
Could add `association :user, factory: :user`. Technically it should be an hmis user, so if needed create a hmis_users factory similar to spec/factories/users.rb